backupstoragelocation.md

```diff
@@ -46,19 +46,25 @@ spec:
     # Optional.
     subscriptionId: my-subscription
 
-    # URI of the blob endpoint of the storage account. 
-    # 
-    # Optional. This will ensure that velero uses the provided URI to communicate to the Storage Account, 
+    # URI of the blob endpoint of the storage account.
+    #
+    # Optional. This will ensure that velero uses the provided URI to communicate to the Storage Account,
     # and it will not try to fetch the Endpoint by making an ARM call.
     # If this field is provided then resourceGroup, subscriptionId can be left empty
     storageAccountURI: my-sa.blob.core.windows.net
 
-    # Boolean parameter to decide whether to use AAD for authenticating with the storage account. 
+    # Boolean parameter to decide whether to use AAD for authenticating with the storage account.
     # If false/ not provided, plugin will fallback to using ListKeys
     #
     # Optional. Recommended.
     useAAD: "true"
 
+    # URI of the AAD endpoint of the storage account.
+    #
+    # Optional. This will ensure that velero uses the provided AAD URI to authenticate to the Storage Account.
+    # Note that useAAD: should be set to "true" in order to use the provided AAD URI
+    activeDirectoryAuthorityURI: my-sa.blob.core.windows.net
```

Comment:
This config should also work for doing snapshots, so please add the config item to the `VolumeSnapshotLocation` and update the code.

Reply:
Thanks, sorry for the delay, ill get to this today